### PR TITLE
docs: Update /v1/node/{node-id} example response

### PR DESCRIPTION
**Pull request by endocrimes**
The nodes api documentation is fairly out of date, here I've updated the
entire response based on a local dev agent, rather than explicitly
adding new fields to bring us up to the current api shape.

### CODE REVIEW

File: website/source/api/nodes.html.md [hunk 1]
```diff
@@ -154,103 +154,106 @@ $ curl \
...
-    "os.signals": "SIGSTOP,SIGTTIN,SIGWINCH,SIGXCPU,SIGXFSZ,SIGIO,SIGKILL,SIGTTOU,SIGINT,SIGHUP,SIGTRAP,SIGALRM,SIGPIPE,SIGURG,SIGABRT,SIGSEGV,SIGIOT,SIGTERM,SIGBUS,SIGPROF,SIGQUIT,SIGTSTP,SIGUSR2,SIGFPE,SIGCONT,SIGILL,SIGSYS,SIGUSR1,SIGCHLD",
-    "cpu.totalcompute": "2200",
+    "nomad.revision": "30da2b8f6c3aa860113c9d313c695a05eff5bb97+CHANGES",
     "driver.raw_exec": "1",
-    "driver.java.version": "openjdk version \"1.8.0_162",
     "kernel.name": "linux",
-    "unique.cgroup.mountpoint": "/sys/fs/cgroup",
-    "driver.docker.volumes.enabled": "1",
-    "cpu.frequency": "2200",
+    "driver.mock": "true",
```
Comment (jippi) [inline website/source/api/nodes.html.md]: bonus point for sorting these keys in the example by name :)

File: website/source/api/nodes.html.md [hunk 2]
```diff
@@ -259,35 +262,110 @@ $ curl \
...
       "Message": "Node registered",
       "Subsystem": "Cluster",
-      "Timestamp": "2018-04-10T23:43:17Z"
+      "Timestamp": "2019-08-26T12:22:50+02:00"
     }
   ],
-  "HTTPAddr": "10.138.0.5:4646",
-  "ID": "f7476465-4d6e-c0de-26d0-e383c49be941",
+  "HTTPAddr": "127.0.0.1:4646",
+  "HostVolumes": {
```
Comment (jippi) [inline website/source/api/nodes.html.md]: this won't be valid until 0.10?

Comment (endocrimes) [reply website/source/api/nodes.html.md]: Correct - `master` is currently the `0.10` dev branch tho :)

File: website/source/api/nodes.html.md (hunk 1 above)
Comment (endocrimes) [reply website/source/api/nodes.html.md]: `jq --sort-keys` to the rescue xD

**Pull request merged**